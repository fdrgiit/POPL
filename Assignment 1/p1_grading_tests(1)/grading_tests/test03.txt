
A123.1000001110000INT


