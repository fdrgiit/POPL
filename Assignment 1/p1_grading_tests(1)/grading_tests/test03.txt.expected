2 35 A123.1000001110000INT 
